entrypoint.py

```diff
@@ -123,7 +123,7 @@
             print(NOTICE
                   + "Failed to push. Going to pull and try again."
                   + ENDC)
-            git.pull()
+            git.pull('--rebase')
```

Comment:
See https://github.com/ponylang/changelog-bot-action/pull/33#issuecomment-690621820

```suggestion
            git.pull(rebase=True)
```